includes/class-sensei.php

```diff
@@ -248,28 +248,16 @@ protected function init() {
 
 		$this->initialize_global_objects();
 
-		$this->maybe_init_email_signup_modal();
-	}
-
-	/**
-	 * Load the email signup modal if we haven't already.
-	 */
-	private function maybe_init_email_signup_modal() {
-		add_action( 'admin_init', array( $this, 'load_email_signup_modal' ) );
 	}
 
 	/**
 	 * Load the email signup modal form.
 	 *
-	 * @access private
+	 * @deprecated 3.1.0 - The modal was removed.
```

Comment:
Just to keep the sandards, I think we could remove the `-`.

Reply:
7f3bf22